redirects-list.json

```
@@ -101,6 +101,7 @@
   "^/doc/dvclive/dvclive-with-dvc$                                                        /doc/start/experiments",
   "^/doc/dvclive/api-reference$                                                           /doc/dvclive/",
   "^/doc/dvclive/get-started$                                                             /doc/start/experiments",
+  "^/doc/dvclive/api-reference/ml-frameworks$                                             /doc/dvclive/ml-frameworks",
```

😅 How do I make:

- https://dvc.org/doc/dvclive/api-reference/ml-frameworks/ANY 

Go to:

- https://dvc.org/doc/dvclive/ml-frameworks/ANY

---

I thought this worked like "replace regex on left with string on right"

---

We have similar usage above.
```suggestion
  "^/doc/dvclive/api-reference/ml-frameworks(/.*)?$                                             /doc/dvclive/ml-frameworks$1",
```